sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -947,13 +948,12 @@ def test_neighbors_badargs():
                       cls, p=-1)
         assert_raises(ValueError,
                       cls, algorithm='blah')
-        nbrs = cls(algorithm='ball_tree', metric='haversine')
-        assert_raises(ValueError,
-                      nbrs.predict,
-                      X)
-        assert_raises(ValueError,
-                      ignore_warnings(nbrs.fit),
-                      Xsparse, y)
```

Comment:
Why don't we keep the above test?

Reply:
Really I don't remember. I will add again